models/user.server.model.js

```diff
@@ -20,7 +22,7 @@ createUser = function(params, done) {
         username = params.username,
         displayName = params.displayName,
         email = params.email,
-        hashedPassword = hashPassword(params.hashedPassword);
+        hashedPassword = hashPassword(params.password);
```

Comment:
```suggestion
        hashSaltedPassword = hashSaltPassword(params.plaintextPassword);
```
If we are going to use a more readable name then why not use hashSalted and plaintextPassword to more clearly define the state of the password in each case